test/framework/curatedpackages.go

```diff
@@ -25,3 +26,17 @@ func WithPackageConfig(t *testing.T, bundleURI, chartName, chartURI,
 		}
 	}
 }
+
+func CheckCuratedPackagesCredentials(t *testing.T) {
+	requiredEnvVars := []string{
+		"EKSA_AWS_REGION",
```

Comment:
Currently we don't require the region flag

Reply:
Ok will remove this.